src/models/NetworkSummaryModel/NdexNetworkProperty.ts

```diff
@@ -1,9 +1,9 @@
 import { IdType } from '../IdType'
-import { ValueTypeName } from '../TableModel'
+import { ValueTypeName, ValueType } from '../TableModel'
 
 export interface NdexNetworkProperty {
   subNetworkId: IdType | null
-  value: string
+  value: ValueType
```

Comment:
I could be mistaken but it seemed that ndex network properties could be of any value type.  Can you confirm if these are really just strings?